scripts/py_matter_idl/matter_idl/zapxml/handlers/handlers.py

```diff
@@ -470,8 +470,11 @@ def __init__(self, context: Context, idl: Idl):
             side=ClusterSide.CLIENT,
             name=None,
             code=None,
+            attributes=[attribute for code,
+                        attribute in self.context.GetGlobalAttributes().items()],
```

Comment:
I am somewhat unsure about the placement - if we want things to be order-independent, we probably should attach global attributes at the time of parsing global attributes as well or have a post processor after "parse all XMLs" is done.

Reply:
Ideally parsing "onoff.xml; global-attributes.xml" should yield the same results as parsing "global-attributes.xml; onoff.xml" if we can do that.

Reply:
I have updated the PR with a few additional changes. Now `configurator::cluster::globalattribute` tags are ignored and a post processing task adds all the `configurator::global` attributes to all the clusters.

As a result the parsing task does not depends on the ordering anymore and I have updated the code in the few places that were relying on `global-attributes.xml` to be the first thing to be parsed.

The additional changes looks big enough that you may want to review it again.